contracts/scripts/ci/generate_uml.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail

```

Reply:
I don't I want to run it in strict mode, because I've seen contract flattening fail due to some weird issues that I do not understand. It failed in the same way on local. I'd rather warn user that this happen than fail whole pipeline. WDYT?

Reply:
In that case, you could catch the errors and wrap a specific code block with `set +e` for example.